### DISCUSSION TITLE
Mac users FYI: hacktribe-gui.exe works with WineStable

### DISCUSSION THREAD

**Discussion by 777paranoia** [bangcorrupt/hacktribe#173] [General]
yo, mac user here, hadn't seen any mention of this (maybe i missed it) but if anyone else on mac is having trouble with the python and terminal stuff for hacking the firmware like i was, just wanted to let everyone know i got the windows executable to work properly using wine. idk about anyone else but the wine stable would crash every time i'd click "browse" for the directories, but it just takes manually typing out them out, and them being formatted relative to your wine C:/ drive

made a lil visual for everyone, my user folder is "studio" but just replace that with whatever yours is. cheers.

![Screen Shot 2022-06-15 at 3 45 58 AM](https://user-images.githubusercontent.com/68776519/173788812-e41cd54b-f0d1-4fe5-9ddb-3336953454eb.png)

**Comment by BKLronin** [comment]
@bangcorrupt 
I can possibly make an mac executabel as I have already set up a docker mac for things like this. Would that be something? Not gonna invest the time if it not gets pulled. I know its difficult with repos and contributers and rights etc.

EDIT: I totally forgot about bs patch so no chance.

**Comment by bangcorrupt** [comment]
> crash every time i'd click "browse"

Can you  post the terminal output from the crash?  I have no issues running it in Wine on Linux.  

There may be a configuration option preventing Wine accessing files outside of its prefix.  It should work if you clone the repo from within Wine.

**Reply by 777paranoia** [reply]
hey man, im running wine thru winestable so not sure about the terminal output; is there a way to view the terminal log of an app thats being run in wine or something? i could send you it that way. 

but im getting this when i click browse, then the following screen when i close out of that window
![Screen Shot 2022-06-21 at 9 02 05 PM](https://user-images.githubusercontent.com/68776519/174927460-6cccf552-ca1a-497a-aa47-a5d5e285f182.png)
![Screen Shot 2022-06-21 at 9 02 23 PM](https://user-images.githubusercontent.com/68776519/174927472-32b7d803-8d22-410d-8c72-595ca6f84afc.png)

**Comment by bangcorrupt** [comment]
> view the terminal log of an app thats being run in wine

Start the app from the terminal.

    wine hacktribe-gui.exe

The `Show Details` button could give some more information.  There is also `hacktribe-app.log` but this probably won't tell us much.
